lib/mayaUsd/render/MaterialXGenOgsXml/ShaderGenUtil.cpp

```diff
@@ -316,7 +316,11 @@ std::string TopoNeutralGraph::gatherChannels(const mx::Input& input)
     }
     const auto& ngInput = interfaceInput ? *interfaceInput : input;
 
+#if MATERIALX_MAJOR_VERSION == 1 && MATERIALX_MINOR_VERSION <= 38
     std::string channelInfo = ngInput.getChannels();
+#else
```

Comment:
Remove the `#else` part. Channels are gone, so we should not see any here. And if we see some, it will be because we forgot to upgrade the document from 1.38 to 1.39, which might be needed in a few places.

Reply:
For this one, do you think `gatherChannels` could just be bypassed entirely, or return an empty string?

Reply:
Yes, please bypass it entirely on 1.39.

Reply:
No worries, I got some time to look at this - instead of passing an empty `channelInfo` around I needed to tweak the `cloneConnection`/`cloneNodeGraphConnection` functions to not have `channelInfo`.